src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,21 +1,41 @@
 import React from 'react';
 
+import { actionCreater } from '../../redux/state';
+
+import ACTION_TYPES from '../../actionTypes';
```

Comment:
Consider grouping this import with other imports of the same level ("../../")

Reply:
I groupted